react/src/components/toggle/SprkToggle.js

```diff
@@ -92,34 +108,61 @@ SprkToggle.propTypes = {
    */
   toggleIconName: PropTypes.string,
   /**
+   * Deprecated - use `triggerText` instead.
    * The title text for the toggle.
    */
-  title: PropTypes.string.isRequired,
+  // TODO: Remove title in issue #1296
+  title: PropTypes.string,
+  /**
+   * The title text for the toggle.
+   */
+  triggerText: PropTypes.string.isRequired,
   /** The content that will show up when the toggle opens. */
   children: PropTypes.node.isRequired,
   /**
-   * Assigned to the `data-id` attribute serving as a unique selector for automated tools.
+   * Assigned to the `data-id` attribute serving as
+   * a unique selector for automated tools.
    */
   idString: PropTypes.string,
   /**
-   * Assigned to the `data-analytics` attribute serving as a unique selector for outside libraries to capture data.
-  */
+   * Assigned to the `data-analytics` attribute serving as
+   * a unique selector for outside libraries to capture data.
+   */
   analyticsString: PropTypes.string,
-  /** Determines if the toggle is open upon loading on the page. */
+  /**
+   * Deprecated - Use `isOpen` instead.
+   * Determines if the toggle is open upon loading on the page.
+   */
+  // TODO: isDefaultOpen in issue #1296
   isDefaultOpen: PropTypes.bool,
-   /**
-   * A space-separated string of classes to add to the outermost container of the component.
+  /** Determines if the toggle is open upon loading on the page. */
```

Comment:
hey @guern1kn , part of the a/c is that they can have it be open or closed whenever.  Lets update this prop description so it mentions that its not just on page load.